CollapseLauncher/XAMLs/MainApp/Pages/HomePage.xaml.cs

```diff
@@ -1794,38 +1793,42 @@ private static int ReadPlaytimeFromRegistry(string RegionRegKey)
 
         }
 
-        private static void SavePlaytimetoRegistry(string RegionRegKey, int value)
+        private static void SavePlaytimeToRegistry(string regionRegistryKey, int value)
         {
             try
             {
-                Registry.CurrentUser.OpenSubKey(RegionRegKey, true).SetValue("CollapseLauncher_Playtime", value, RegistryValueKind.DWord);
+                Registry.CurrentUser.OpenSubKey(regionRegistryKey, true).SetValue("CollapseLauncher_Playtime", value, RegistryValueKind.DWord);
             }
             catch (Exception ex)
             {
                 LogWriteLine($"Playtime - There was an error writing to registry. \n {ex}");
             }
         }
 
-        private async static void StartPlaytimeCounter(string oldRegionRegistryKey, Process proc, PresetConfigV2 gamePreset)
+        private async void StartPlaytimeCounter(Process proc, PresetConfigV2 gamePreset)
         {
-            int saveFrequencyinSeconds = 60;
+            int currentPlaytime = ReadPlaytimeFromRegistry(gamePreset.ConfigRegistryLocation);
 
-            int currentPlaytime = ReadPlaytimeFromRegistry(oldRegionRegistryKey);
-            int elapsedSeconds = 0;
+            DateTime begin = DateTime.Now;
 
-            var inGameTimer = new DispatcherTimer
-            {
-                Interval = TimeSpan.FromSeconds(1)
-            };
-            inGameTimer.Tick += (o, e) =>
+            System.Timers.Timer inGameTimer = new System.Timers.Timer();
```

Comment:
I suggest to consider using ``using`` statement since ``Timer`` is disposable.
Also,  I think it's better to include the ``System.Timers`` namespace to the code instead of calling it explicitly.

Reply:
```System.Timers.Timer``` conflicts with ```System.Threading.Timer``` and as it's the only use of it in the entire file, I just called it explicitly.
About ```using```, I've added it.